docs/source/contributing/developing.adoc

```diff
@@ -9,15 +11,15 @@ toc::[]
 [[developing-overview]]
 == Overview
 
-The following sections describe how to build and test Minishift.
+The following sections describe how to build and test {project}.
```

Comment:
Do we really need this? The whole contributing topic seems pointless for the CDK. Would you for the CDK docs not ignore this altogether anyway? I guess there might be a case for consistency.

Reply:
Yes, the CDK documentation would ignore this completely -- all of the changes made were entirely for the sake of consistency throughout the documentation.

This also means not having to think about whether you should use "Minishift" or "{project}" -- it's always "{project}" unless we're referring to the binary. If that's not desired, I'd be happy to do a more thorough PR which only replaces "Minishift" as-needed for CDK docs.

Reply:
To elaborate, the main point here is to provide flexibility and consistency. The `minishift` binary never changes, but the project name does. Having a split between "do I write {project} or Minishift here" when writing the documentation adds some overhead and introduces the potential for errors.

As an example, writing under **contributing/** followed by some edits to a document in **using/** could easily result in accidentally using "Minishift" where {project} is more appropriate. In addition to this, if we're using both at varying times in varying places, it becomes easy to accidentally overlook the mistake.

Reply:
> This also means not having to think about whether you should use "Minishift" or "{project}" -- it's always "{project}" unless we're referring to the binary.

Sure, makes sense. I was expecting that this was the idea behind it. Just trying to highlight the pros and cons and potential issues.

As you say, if we are not going to use "{project}" consistently, people will always have a hard time to know whether to use the variable or the literal 'Minishift'